<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>838</width>
    <height>557</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>838</width>
    <height>651</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>0</y>
      <width>491</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="mRegistation">
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">selection-color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>Registation</string>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>16</height>
        </size>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mLogin">
       <property name="text">
        <string>Login</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bHelp">
       <property name="text">
        <string>Help ?</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>130</y>
      <width>461</width>
      <height>311</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="vUserdisplay"/>
   </widget>
   <widget class="QLabel" name="label_Hi">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>270</y>
      <width>160</width>
      <height>241</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutControl">
     <item>
      <widget class="QPushButton" name="bViewBooks">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>viewBooks</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>90</y>
      <width>461</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Search for</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditSearch">
       <property name="styleSheet">
        <string notr="true">border-left-color: rgb(255, 255, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bsearchBooks">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Go !</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="bFindAdvance">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>60</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Advance</string>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>label</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label_Hi</zorder>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>bFindAdvance</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>838</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
